communication/marketing-team/blog-guidelines.md

```diff
@@ -0,0 +1,63 @@
+# Upstream Marketing Blog Guidelines
+
+This initiative falls under the [Upstream Marketing Charter](./CHARTER.md).
+
+We are looking for Kubernetes-curious community members who are **interested in writing** and **care about getting the word out** to our huge community of users, developers, and contributors of all types. Here's how to get involved.
+
+## Requested Content
+
+We are looking for: 
+
+* TODO << Request specific types of content
+
+Other types of content, like Kubernetes capabilities and tutorials, are better suited for the [SIG-Docs blogging initiative](https://github.com/kubernetes/community/blob/77aa44ba8a506b2d8a7110ff96f545c36db906f3/sig-docs/blog-subproject/README.md).
+
+## Submit a Post
+
+The quickest way to get involved is to let the team in [#sig-contribex](https://kubernetes.slack.com/archives/C1TU9EB9S) that you have an idea for an article.
+
+For now, our official process is to use [SIG-Doc's system](https://github.com/kubernetes/community/blob/77aa44ba8a506b2d8a7110ff96f545c36db906f3/sig-docs/blog-subproject/README.md).
+
+## Blogger Expectations, Responsibilities, and Info
+
+Anyone is welcome to contribute when they have time. 
+
+If you would like to be listed as a member of the team, here are the expectations:
+
+1. Be prepared to write one blog a quarter and participate in edits to other articles. The time commitment is typically 5-10 hours per quarter depending on the number of blog posts in the review queue.
+2. Bloggers are expected to attend at least one upstream marketing team meeting a month or check-in to remain active. 
+3. Remain non-partial: if you receive a request to write about a project, an individual, or a group of people from your employer, you should ask a non-partisan blogger to write it.
```

Comment:
impartial?

Reply:
Much better word. I saw non-partisan and couldn't think of a what to use so I left it there. Fixed in the commit that's coming.